Keep forms in bold even when active.
By: yozohida@gmail.com
---
 src/viewer/text/link.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

Index: elinks/src/viewer/text/link.c
===================================================================
--- elinks.orig/src/viewer/text/link.c	2017-11-01 03:00:11.082815434 +0100
+++ elinks/src/viewer/text/link.c	2017-11-01 03:02:36.411397296 +0100
@@ -184,7 +184,8 @@
 	if (doc_opts->active_link.underline)
 		template_.attr |= SCREEN_ATTR_UNDERLINE;
 
-	if (doc_opts->active_link.bold)
+	/* keep forms in bold */
+	if (doc_opts->active_link.bold || link->type != LINK_HYPERTEXT)
 		template_.attr |= SCREEN_ATTR_BOLD;
 
 	if (doc_opts->active_link.enable_color) {
